src/main/js/controls/Pie/PieConfig.js

```diff
@@ -21,6 +21,42 @@ export const processInput = (input, config) => {
     config.showLegend = getDefaultValue(input.showLegend, true);
     return config;
 };
+/**
+ * Checks if panning is enabled or not
+ *
+ * @private
+ * @param {object} config - config object used by the graph.
+ * @returns {boolean} returns true of panning enabled else false.
+ */
+export const isPanningModeEnabled = (config) => {
```

Comment:
Why does Pie chart need panning?

Reply:
I will remove it

Reply:
Why not just pass the constant with enabled transition `D3_TRANSITION_PROPERTIES_ENABLED`, when you are calling `constants.d3Transition`?

Reply:
Do you mean to hardcode "D3_TRANSITION_PROPERTIES_ENABLED" for pie at all the places where we are calling transition?